CHANGES.rst

```diff
@@ -0,0 +1,35 @@
+Changelog
+---------
+
+0.1 (2015-08-13)
+~~~~~~~~~~~~~~~~
+
+First release.
+
+* core contributors, in alphabetical order:
+
+  * Eric Battenberg (@ebattenberg)
+  * Sander Dieleman (@benanne)
+  * Daniel Nouri (@dnouri)
+  * Eben Olson (@ebenolson)
+  * Aäron van den Oord (@avdnoord)
+  * Colin Raffel (@craffel)
+  * Jan Schlüter (@f0k)
+  * Søren Kaae Sønderby (@skaae)
+
+* extra contributors, in chronological order:
```

Comment:
You can add my real name: Gregory Sanders  

Before I forget: I've become busy with other stuff. If someone just wants to poach the PreLU implementation go ahead and take it. Don't have time to finish.